src/scythe-bidder/create-room.tsx

```diff
@@ -59,6 +64,31 @@ export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
                   ))}
               </Select>
             </Form.Item>
+            <Form.Item>
+              <Select<string>
+                value={gameType}
+                onChange={(value) => {
+                  setGameType(value);
+                  if (value === SCYTHE_BASE){
+                    if (numPlayers > MAX_PLAYERS_BASE){
+                      setNumPlayers(MAX_PLAYERS_BASE);
+                      notification["warning"]({
```

Comment:
nit: I prefer `notification.warning`. Also, maybe change the "message" to "Warning"?

Reply:
I've made this change.